### ISSUE TITLE
rustdoc JS on doc.r-l.o is not working on nightly

### ISSUE THREAD

**Issue by Mark-Simulacrum** [rust-lang/rust#58290]
```
SyntaxError: missing ) after argument list[Learn More] main.js:1:38574
ReferenceError: initSearch is not defined[Learn More] search-index.js:8:1
    <anonymous> https://doc.rust-lang.org/nightly/search-index.js:8
```

cc @GuillaumeGomez @QuietMisdreavus

**Comment by RalfJung** [comment]
Is this also the reason that rustdoc search on https://doc.rust-lang.org/nightly/std/index.html is broken?

**Comment by Mark-Simulacrum** [comment]
Yes, this will prevent any JS from working across nightly-built docs until this is fixed.

**Comment by kennytm** [comment]
This has been fixed by #58294, and a nightly has been released. Closing.
